cert/aziot-certd/Cargo.toml

```diff
@@ -5,9 +5,9 @@ authors = ["Azure IoT Edge Devs"]
 edition = "2018"
 build = "build/main.rs"
 
+
```

Comment:
I did them explicitly, because the `[dependencies]` section has one-line gaps between groups of deps, so it seems nicer (?) to have two-line gaps between the sections themselves.

Reply:
Uhh, sure? In that case, you might want to skim through the rest of the TOML files and keep them consistent.

Either way though, not sure if it makes sense to have these changes in this PR, but s'aight.

inb4 tomlfmt when??

Reply:
I've fixed the ones that are being edited in this PR. We can fix the rest when we edit them.